### PR TITLE
Update readme code and ui glitch

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Link: https://phabricator.wikimedia.org/T340680

additional a quick ui glitch fix

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -143,7 +143,7 @@ In the admin site, go to “Appearance / customize”, then select the “Additi
 .wmf-wp-with-preview::after {
     content: '';
 }
```
Comment (medied) [inline readme.txt:145]: Now that the new design is live I don't think we are making use of `::after` , I think that's a leftover from previous styling

Comment (medied) [reply readme.txt:145]: We should probably update this in [README.md](https://github.com/wikimedia/wikipediapreview-wordpress/blob/main/README.md) too

Comment (hueitan) [reply readme.txt:145]: checkout the latest css change here
https://github.com/wikimedia/wikipedia-preview/pull/174

Comment (medied) [reply readme.txt:145]: Oh got it! Nevermind then this looks good

**Pull request merged**